pom.xml

```diff
@@ -116,6 +116,11 @@ OF THE POSSIBILITY OF SUCH DAMAGE.
       <artifactId>guava</artifactId>
       <version>31.1-jre</version>
     </dependency>
+    <dependency>
+      <groupId>org.cactoos</groupId>
```

Comment:
@volodya-lombrozo can't you use Guava instead? It's already in the list of dependencies and provides the same functionality of turning an `Iterable` to a `List`.

Reply:
@yegor256 Done. Could you have a look, again, please?